test/integration/red_test_python_aws_s3.go

```diff
@@ -35,6 +35,21 @@ func testPythonAWSS3(t *testing.T) {
 	waitForTestComponentsNoMetrics(t, address+"/health")
 	waitForTestComponentsNoMetrics(t, localstackAddress)
 
+	// Wait for /health to appear in jaeger
```

Comment:
I made a small fix to the AWS S3 test since it failed on me with createBucket missing. I checked the logs and we caught the application too late, we saw the response of the `PUT  /obi-bucket` but not the actual request. I made us wait calling `/health` until we see at least one jaeger trace with `/health`. Then we go into verifying the buckets.

Reply:
Thanks! I thought the issue was on `PUT /bucket/item` 🙃 , let's hope this fixes it